_data/finance.yml

```diff
@@ -25,8 +25,9 @@ websites:
     - name: FreeAgent
       url: https://www.freeagent.com
       img: freeagent.png
-      twitter: freeagent
-      tfa: No
+      tfa: Yes
+      software: Yes
+      docs: https://www.freeagent.com/support/kb/getting-started/two-step-verification
```

Comment:
I missed the change, just saw the git commit headline of `Remove twitter link`. Sorry, everything's perfect now!